optimizely/entities/user_context_test.go

```diff
@@ -1,6 +1,7 @@
 package entities
```

Comment:
Looks like I forgot to add the license header, can you include it please

Reply:
addressed